CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2021-05-25 unftp v0.12.10
+
```

Comment:
Would be good to also mention the fix of the logger whether in this PR or after.